test/jdk/javax/swing/JMenuItem/4654927/bug4654927.java

```diff
@@ -84,10 +89,12 @@ public void run() {
             // test mouse drag
             point = Util.getCenterPoint(menu);
             robot.mouseMove(point.x, point.y);
+            robot.delay(250);
```

Comment:
This delay may help. I saw that moving mouse takes a bit of time on macOS, the following code heavily depends on the mouse being in the expected position. You may assert that mouse cursor is at the expected position.

On the other hand, mouse press and further movements shouldn't precede the first mouse move event. If it happens, it could be a bug in macOS. Thus I can't see how this can lead to the test failure.

Which case does the test fail by the way? Is it the first or the second?